data-serving/data-service/src/controllers/case.ts

```diff
@@ -44,17 +44,29 @@ export const list = async (req: Request, res: Response): Promise<void> => {
               $text: { $search: parsedSearch.fullTextSearch },
           }
         : {};
-    // Do a fetch of documents and another fetch in parallel for total documents
-    // count used in pagination.
+    // Fill in keyword filters.
     try {
+        let q = Case.find(query);
```

Comment:
q isn't the most clear variable name now that it means more than query

Reply:
sorry, can't get rid of my golang habits that easily ;) fixed.